src/main/java/io/fabric8/jenkins/openshiftsync/PodTemplateUtils.java

```diff
@@ -258,6 +268,26 @@ protected static boolean isReservedPodTemplateName(String name) {
       return results;
   }
 
+  protected static PodTemplate updatePodTemplateEnvVars(HashMap<String,String> map, PodTemplate podTemplate){
+      List<PodEnvVar> envVars = podTemplate.getEnvVars();
+
+      for (String key : map.keySet()){
+          PodEnvVar envVar = new PodEnvVar(key, map.get(key));
+          envVars.add(envVar);
+      }
+      podTemplate.setEnvVars(envVars);
+      return podTemplate;
+  }
+
+  protected static PodTemplate setProxyConfigToPodTemplate(PodTemplate podTemplate){
+    HashMap envMap = new HashMap();
```

Comment:
it is preferable to use interface rather than implementation:
Map<K,V> map = new HashMap<>();

Reply:
done